tests/mocks/LoanManagerHarness.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.19;
+
```

Comment:
Import interfaces before the contract to enhance readability by keeping the most important sections closer to the main contract declaration.